.github/workflows/manual-release.yml

```diff
@@ -11,5 +11,6 @@ jobs:
   manual-release:
     uses: ./.github/workflows/release.yml
     with:
-      token: ${{ secrets.GITHUB_TOKEN }}
       tag: ${{ inputs.tag }}
+    secrets:
+      token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Because it is a workflow, instead of a composite, you shouldn't need to pass. The workflow can have its own permissions.

Reply:
https://github.com/launchdarkly/dotnet-logging-adapter-nlog/blob/3c92a9e0ad1c64a2a327ed2bffab3ca25e3906f8/.github/workflows/release-please.yml#L27

Reply:
```
When a reusable workflow is triggered by a caller workflow, the github context is always associated with the caller workflow. The called workflow is automatically granted access to github.token and secrets.GITHUB_TOKEN. For more information about the github context, see "[Contexts](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context)."
```

Reply:
Yea, I see that `secrets` is automatically available. This does work though, but I'll refactor to remove it.